frontend/src/components/formActions.tsx

```diff
@@ -57,16 +56,16 @@ const FormActions: React.FC<IProps> = ({ handleClick }: IProps) => {
           <>
             <div className={classes.actionsWrapper}>
               <Typography component="p" className={classes.p}>
-                {t("agreeToOur")}
+                I am not a minor and accept and agree to
+                <a href="/terms-of-use" className={classes.a}>
```

Comment:
I would add a `target="_blank"` so that the user doesn't lose their form progress by clicking the link and leaving the page just before submitting.